frontend/awx/access/organizations/OrganizationPage/OrganizationPage.tsx

```diff
@@ -66,7 +66,7 @@ export function OrganizationPage() {
       },
     ];
     return itemActions;
-  }, [deleteOrganizations, history, t]);
+  }, [deleteOrganizations, nagivate, t]);
```

Comment:
should be 'navigate'

Reply:
`useNavigate` hook is set to the variable that is misspelled  as`nagivate`